weed/storage/volume_write.go

```diff
@@ -80,16 +80,18 @@ func (v *Volume) Destroy(onlyEmpty bool) (err error) {
 		}
 	}
 	v.doClose()
-	removeVolumeFiles(v.DataFileName())
-	removeVolumeFiles(v.IndexFileName())
+	removeVolumeFiles(v.DataFileName(), afterEc)
+	removeVolumeFiles(v.IndexFileName(), afterEc)
 	return
 }
 
-func removeVolumeFiles(filename string) {
+func removeVolumeFiles(filename string, afterEc bool) {
```

Comment:
`afterEc` only shows when. Please change to a more meaningful name for what it does.